src/endpoints/auth/outputs/forgot-email.output-schema.ts

```diff
@@ -0,0 +1,7 @@
+export const forgotEmailOutputSchema = {
```

Comment:
You're using such a schema multiple times across the project. Maybe it's better to create a single universal StatusOutputSchema?